index.html

```diff
@@ -223,7 +259,7 @@ <h3 class="skill-img ab"> <img src="img/skill55.png" alt=""> Language</h3>
         </div>
         <div class="card-skill">
             
-            <h3 class="skill-img ab"> <img src="img/framwork56.png" alt=""> Framework</h3>
+            <h3 class="skill-img mab"> <img src="img/framwork56.png" alt=""> Framework</h3>
```

Comment:
- [ ] Kindly put the lists on the same line level, also the content of each list should be aligned to the left and not centered 👍 

## Your design:
![image](https://user-images.githubusercontent.com/50721479/215283578-9316737b-4224-4fc3-a5fe-3fb1afa612b8.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/215283591-57de7847-466f-446d-9786-506a5b1aa3a6.png)